inductiva/simulators/fenicsx.py

```diff
@@ -15,25 +15,44 @@ def __init__(self):
     def run(
         self,
         input_dir: types.Path,
-        mesh_filename: str,
+        geometry_filename: str,
         bcs_filename: str,
         material_filename: str,
+        global_refinement_factor: float = 1.0,
+        local_refinement_factor: float = 0.0,
         machine_group: Optional[resources.MachineGroup] = None,
         run_async: bool = False,
     ) -> tasks.Task:
         """Run the simulation.
 
         Args:
-            mesh_filename: Mesh filename.
+            geometry_filename: Geometry filename.
             bcs_filename: Boundary conditions filename.
             material_filename: Material filename.
+            global_refinement_factor (float): The refinement factor for global
```

Comment:
This should be called `meshing_global_refinement_factor` or something similar, to make it explicit that this is connected to the meshing process.